packages/fxa-settings/src/components/Ready/en.ftl

```diff
@@ -0,0 +1,7 @@
+## Ready component
+
+ready--confirmation = Your password has been reset
```

Comment:
nit: not sure if we use bem style naming w/ double dash elsewhere.
My quick grep shows these are the only 4 instances in the .ftl files:
```sh
git grep -EIn "\-\-" packages/**/*.ftl | cat

# packages/fxa-settings/src/components/Ready/en.ftl:3:ready--confirmation = Your password has been reset
# packages/fxa-settings/src/components/Ready/en.ftl:7:ready--use-service = You’re now ready to use { $serviceName }
# packages/fxa-settings/src/pages/ResetPasswordConfirmed/en.ftl:3:pw-reset-confirmed--confirmation = Your password has been reset
# packages/fxa-settings/src/pages/ResetPasswordConfirmed/en.ftl:7:pw-reset-confirmed--use-service = You’re now ready to use { $serviceName }
```